integration_tests/mockApis/utils.ts

```diff
@@ -54,7 +54,7 @@ const errorStub = (fields: Array<string>, pattern: string, method: 'PUT' | 'POST
 const bedspaceConflictResponseBody = (entityId: string | LostBed, entityType: 'booking' | 'lost-bed') => ({
```

Comment:
Again, `entityType` should surely use an existing type here? Also what's going on with `entityId`? That's an odd union 🤔

Reply:
That's a very strange union as LostBed is an object.
I'm not touching it in this ticket though.